robottelo/ui/role.py

```diff
@@ -85,3 +85,29 @@ def add_permission(self, role_name, resource_type=None,
             self.click(tab_locators['roles.tab_location'])
             self.configure_entity(location, FILTER['filter_loc'])
         self.click(common_locators['submit'])
+
+    def get_resources(self, role_name):
```

Comment:
We need to re-implement `get_permissions` too as we passing `resource_type` there. We do not need search for permissions, but for resource, so need almost the same fix in approach as we should have for `get_resources`